libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/Plugins/SFNetworkPlugin/SFNetworkPlugin.m

```diff
@@ -57,18 +58,27 @@ - (void) pgSendRequest:(CDVInvokedUrlCommand *) command
     NSString* path = [argsDict nonNullObjectForKey:kPathArg];
 
     NSDictionary* queryParams = [[NSDictionary alloc] init];
+    NSDictionary<NSString*, NSString*>* headerParams = [argsDict nonNullObjectForKey:kHeaderParams];
```

Comment:
> This is not the artifact of old functionality.

Ah, that was me misreading the diff.  It looked like setting the value was new as of this request.  I missed it below.

> I don't see a reason to single it out and treat it as a separate parameter.

My reasoning is that, in a world where we're specifying arbitrary content as input, it makes more sense for the value not to be buried in another collection of properties as an afterthought.  Indeed, that's why Content Type is explicitly called out as a parameter to `SFRestRequest`'s initializer.  We can leave it in a collection, but it feels a bit removed from intention in that use case, to me.